<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsWindow</string>
  </property>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>127</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>gridX:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxGridX">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>gridY:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxGridY">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>186</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLoad">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="boxLoad">
        <property name="toolTip">
         <string>&lt;Return&gt;</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSave">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="boxSave">
        <property name="toolTip">
         <string>&lt;Return&gt;</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>57</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLog">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>noLog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkNoLog">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>172</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>start_Bunnies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxStartBunnies">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>foodCount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxFoodCount">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>food_duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxFoodDuration">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>185</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>max_colony_size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxMaxColonySize">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>infection_prob</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxInfectionProb">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>sleep_time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxSleepTime">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>maxHunger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="boxMaxHunger">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>boxGridX</tabstop>
  <tabstop>boxGridY</tabstop>
  <tabstop>checkLoad</tabstop>
  <tabstop>boxLoad</tabstop>
  <tabstop>checkSave</tabstop>
  <tabstop>boxSave</tabstop>
  <tabstop>checkLog</tabstop>
  <tabstop>checkNoLog</tabstop>
  <tabstop>boxFoodCount</tabstop>
  <tabstop>boxMaxColonySize</tabstop>
  <tabstop>boxInfectionProb</tabstop>
  <tabstop>boxStartBunnies</tabstop>
  <tabstop>boxSleepTime</tabstop>
  <tabstop>boxFoodDuration</tabstop>
  <tabstop>boxMaxHunger</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
